indico/modules/events/contributions/templates/management/fields_dialog.html

```diff
@@ -0,0 +1,101 @@
+            <li>
+                <a data-ajax-dialog class="js-action-button"
+                   data-href="{{ url_for('.create_field', event, field_type=form_type.name) }}"
+                   data-title="{% trans %}Add custom field{% endtrans %}">
+                    {{ form_type.friendly_name }}</a>
+            </li>
+            {% endfor %}
+        </ul>
+    </div>
+
+    {% set description_item = { 'title': _("Abstract description") } %}
+    {% call(item) sortable_list([description_item], draggable=false, invisible_handle=true,
+                                id='contributions-fixed-fields') %}
+        <a class="icon-edit js-action-button" data-ajax-dialog
+           data-href="{{ url_for('.manage_description_field', event) }}"
+           data-title="{% trans %}Abstracts description{% endtrans %}"></a>
+        <a class="js-delete-contribution-field icon-remove disabled"
+           title="{% trans %}This field cannot be deleted{% endtrans %}"
+           data-qtip-position="right"></a>
+    {% endcall %}
+
+    {% if custom_fields %}
+        {% call(custom_field) sortable_list(custom_fields, id="contributions-custom-fields") %}
+            <a class="icon-edit js-action-button" data-ajax-dialog
+               data-href="{{ url_for('.manage_field', event, custom_field) }}"
+               data-title="{% trans %}Edit custom field{% endtrans %}"></a>
+            <a class="js-delete-contribution-field icon-remove"
+               data-method="POST"
+               data-href="{{ url_for('.delete_field', event, custom_field) }}"
+               data-confirm="{% trans %}Do you really want to delete this field? Data stored in this field will be lost.{% endtrans %}"
+               data-title="{% trans %}Confirm deletion{% endtrans %}"></a>
+        {% endcall %}
+    {% else %}
+        {%- call message_box('info') %}
+            {%- trans %}There are no custom contribution fields yet.{% endtrans %}
+        {%- endcall %}
+    {% endif %}
+
+    <div class="toolbar">
+        <div class="group right">
+            <button class="i-button big" data-button-back>{% trans %}Close{% endtrans %}</button>
+        </div>
+    </div>
+</div>
+
+<script>
+    (function() {
+        'use strict';
+
+        var $dialog = $('.manage-contribution-fields');
+
+        $dialog.dropdown();
+
+        $dialog.on('indico:confirmed', '.js-delete-contribution-field', function(evt) {
+            evt.preventDefault();
+            var $this = $(this);
+            $.ajax({
+                url: $this.data('href'),
+                method: $this.data('method'),
+                complete: IndicoUI.Dialogs.Util.progress(),
+                error: handleAjaxError,
+                success: function () {
+                    $dialog.trigger('ajaxDialog:reload');
+                }
+            });
+        });
+
+        $dialog.on('ajaxDialog:closed', '.js-action-button', function(evt, data) {
+            evt.preventDefault();
```

Comment:
Is that actually needed? If not I'd change it to `stopPropagation` since you have a dialog inside a dialog and having it bubble to the parent dialog seems odd. If it's needed I'd simply do both